contracts/PermissionedNodeRegistry.sol

```diff
@@ -56,6 +56,7 @@ contract PermissionedNodeRegistry is
     //mapping of operator Id and nextQueuedValidatorIndex
     mapping(uint256 => uint256) public override nextQueuedValidatorIndexByOperatorId;
     mapping(uint256 => uint256) public socializingPoolStateChangeBlock;
+    mapping(uint256 => address) public pendingRewardAddressByOperatorId;
```

Comment:
rename it to rewardAddressChangeRequestByOperatorId

Reply:
The value stored is a address. The above name seems as if some request struct is stored.
How about `proposedRewardAddressByOperatorId` ?